guess_the_outcome_rnn/Readme.md

```diff
@@ -0,0 +1,37 @@
+# RNN: Guess the outcome of a blackbox
+
+## Problem
+Consider a consistent program as a black-box, assume: 
+- For same inputs the result will always be the same (there is no impact of state or time)
+- The program may throw exceptions which can be considered as an output.
+- Many inputs may lead to same output.
+- Type(s) of input(s) are known.
+
+Using this system as a dataset, try to train a Recurrent Neural Network which is able to 
+guess the output.
```

Comment:
Since this is an exercise and not a problem, we might specify the program upfront (with changes in the problem specification) and make it lean towards a certain network type (e.g. rnn here) and prod possible issues in training those.